doc/api/buffer.md

```diff
@@ -152,7 +158,7 @@ console.log(buf.toString('base64'));
 
 The character encodings currently supported by Node.js include:
 
-* `'ascii'` - for 7-bit ASCII data only.  This encoding method is very fast and
+* `'ascii'` - for 7-bit ASCII data only.  This encoding is very fast and
   will strip the high bit if set.
```

Comment:
May want to remove the part about being "very fast". It's faster than `'utf8'` for sure, but not as fast as `'latin1'`.